index.html

```diff
@@ -0,0 +1,202 @@
+                </div>
+            </div>
+            <div class="work-details-row3">
+                <div class="wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+
+                <div class="wdr32">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr33">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+
+
+        </div>
+
+    </div>
+    <!------------------about me------------------------------------------------>
+    <div class="aboutme-container">
+        <div class="aboutme-row1">
+            <div class="aboutme-left">
+                <h4>About me</h4>
+                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
+                    experience! If you like
+                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p class="gr">Get My Resume</p>
+            </div>
+            <div class="aboutme-right">
+                <img src="./images/aboutme.png">
+            </div>
+
+        </div>
+        <div class="hline">
+            <hr>
+        </div>
+        <div class="aboutme-row2">
+            <div class="aboutme-row2-left">
+                <h3><img src="./images/Rectangle 55.png">Language</h3>
+                <h6>c</h6>
+                <h6>C++</h6>
+                <h6>Javascript</h6>
+            </div>
+            <div class="aboutme-row2-middle">
+                <h3><img src="./images/Rectangle 56.png">Frame works</h3>
+                <h6>React</h6>
+                <h6>django</h6>
+                <h6>Ruby on Rails</h6>
+            </div>
+            <div class="aboutme-row2-right">
+                <h3><img src="./images/Rectangle 57.png">Skills</h3>
+                <h6>Github</h6>
+                <h6>Web Development</h6>
+                <h6>Coding</h6>
+            </div>
+        </div>
+
+    </div>
```

Comment:
- [ ] As mentioned before good job separating works and about me :+1:, but it would be better if you use `<section>` tag instead `<div>` for wrapping 'about me' because besides being semantically correct it would improve the user experience :100: .

- [ ] As the template shows you should increase the size of 'About me' title and check the margins :+1: 

**Figma template**

![Screenshot from 2023-01-17 05-31-57](https://user-images.githubusercontent.com/105117832/212876537-7f24c00c-8692-4ce2-9224-74ab7d1df92b.png)

**Your design**

![Screenshot from 2023-01-17 05-37-43](https://user-images.githubusercontent.com/105117832/212877379-cfc11833-b702-4018-9da3-53224d60cfa2.png)


- [ ] Finally, while you did a wonderful job styling 'about me section' you should also center the technology inside the categories :+1: 

**Figma template**

![Screenshot from 2023-01-17 05-35-00](https://user-images.githubusercontent.com/105117832/212877124-ceb2d75f-6c80-4287-81f0-1330a9c98abe.png)

**Your design**

![Screenshot from 2023-01-17 05-37-28](https://user-images.githubusercontent.com/105117832/212877318-a5388faa-5df7-4a69-ac8b-bb9c52995825.png)